src/components/SkillPage/SkillListing.js

```diff
@@ -177,7 +177,7 @@ class SkillListing extends Component {
         console.log(skill_ratings);
         this.setState({
             skill_ratings: ratings_data,
-            avg_rating: parseInt(skill_ratings.avg_star, 10),
+            avg_rating: parseFloat(skill_ratings.avg_star),
```

Comment:
I suggest you should round it off to 1 or 2 decimal places.

Reply:
Yes it should be taken care of at the server and @akshatnitd has already opened an issue here - https://github.com/fossasia/susi_server/issues/801